jkube-kit/build/api/src/main/java/org/eclipse/jkube/kit/build/api/auth/handler/FromConfigRegistryAuthHandler.java

```diff
@@ -24,31 +24,31 @@
  * @author roland
  */
 public class FromConfigRegistryAuthHandler implements RegistryAuthHandler {
-    private final RegistryAuthConfig registryAuthConfig;
```

Comment:
This class was completely reformatted and is really hard to track the changes (at least in the GH interface).
Could you please revert the reformatting so that it's clearer for us to see what was changed here?